packages/form-system/stories/05-creating-a-custom-field.stories.mdx

```diff
@@ -0,0 +1,89 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { render } from '@lion/core';
+import '@lion/input/lion-input.js';
+import { Validator } from '@lion/validate';
+import './helper-wc/h-output.js';
+
+<Meta title="Form Fundaments/Creating a Custom Field" />
+
+# Creating a custom field
+
+Custom fields can be created in just a few steps. All you need is an interaction element
+(like for instance a slider, a listbox or a combobox) and connect it to the [Field](../README.md)
```

Comment:
The second sentence isn't very clear:
`All you need is an interaction element and connect it to the Field functionality`.

Could we rephrase this?

Reply:
But I think especially most people don't know what an interaction element is, right ? :) 
I would rather elaborate on what that is?